database/migrations/2023_09_25_113832_create_tours_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    protected $connection = 'tours_db';
+
+    public function up(): void
+    {
+        Schema::connection($this->connection)->create('tours', function (Blueprint $table) {
```

Comment:
With the `$connection` property set above, you shouldn't need to pass the connection variable to the `Schema::connection()` method here. It should pull that behind the scenes.